Orange/widgets/data/owpreprocess.py

```diff
@@ -307,10 +307,12 @@ def filterSettingsClicked(self):
 
     def filterByClicked(self):
         self.setFilter0(self.filter_buttons.checkedId())
+        self.edited.emit()
```

Comment:
This line is redundant. `setFilter0()` emits the signal, if `self.filter0` has changed.